libs/dh/esett/feature-outgoing-messages/src/lib/filters/dh-filters.component.ts

```diff
@@ -68,6 +68,10 @@ type Filters = FormControls<DhOutgoingMessagesFilters>;
       watt-dropdown {
         width: auto;
       }
+
+      .documentId-field {
+        width: 20rem;
+      }
```

Comment:
nitpick: doesn't look like this is used.

Reply:
Ohh, you are right, this was used for the textbox, just forgot to remove it.

Reply:
Fixed